web/src/lib/components/elements/buttons/skip-link.svelte

```diff
@@ -7,24 +7,39 @@
      * Target for the skip link to move focus to.
      */
     target?: string;
+    /**
+     * Text for the skip link button.
+     */
     text?: string;
+    /**
+     * Breakpoint at which the skip link is visible. Defaults to always being visible.
+     */
+    breakpoint?: 'sm' | 'md' | 'lg' | 'xl' | '2xl';
   }
 
-  let { target = 'main', text = $t('skip_to_content') }: Props = $props();
+  let { target = 'main', text = $t('skip_to_content'), breakpoint }: Props = $props();
 
   let isFocused = $state(false);
 
   const moveFocus = () => {
     const targetEl = document.querySelector<HTMLElement>(target);
     targetEl?.focus();
   };
+
+  const getBreakpoint = () => {
+    if (!breakpoint) {
+      return '';
+    }
+    return `hidden ${breakpoint}:block`;
```

Comment:
You can't dynamically compute tailwind classes like this. Tailwind scans the source file for class names and patterns and uses the list it finds to create the stylesheet dynamically. You need to have the complete text `sm:block`, etc in the file somewhere.